ros2/cc_defs.bzl

```diff
@@ -92,6 +95,7 @@ def _ros2_cpp_exec(target, name, ros2_package_name = None, set_up_ament = False,
     sh_target = native.sh_test if is_test else native.sh_binary
     sh_target(
         name = name,
+        size = size,
```

Comment:
Not sure this will work with a sh_binary. Can you double-check? An alternative is to create a separate kwargs-like dict that would be empty for sh_binary and have size for sh_test. That dict you would add to the args instead of the size directly.

Reply:
Looks like sh_binary is fine with it, although tbh I can't immediately find an explaanation for it in https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/bazel/rules/sh/BazelShBinaryRule.java

I added a print in just before the sh_target:
```starlark
    if not is_test:
        print("{} is a {} with size={}".format(name, sh_target, size))
```
and it will print
```
DEBUG: (...)/rules_ros2/ros2/cc_defs.bzl:97:14: plugin_tests_binary is a <built-in rule sh_binary> with size=None
```